english/bob-barker/phone-a-friend/phone-a-friend.md

```diff
@@ -0,0 +1,3 @@
+"Sorry bud, wrong show.," replies Bob sarcastically.
+
+Your ears are full of gooey marshmallow, so you ignore Bob's turgid response and reach for your flip-phone and begin to phone the first person that comes to mind....
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.